### ISSUE TITLE
Console not able to handle keys

### ISSUE THREAD

**Issue by tcpie** [crosire/scripthookvdotnet#658]
Open up the console, press "V" -> nothing happens. Same thing with the " key. Makes it pretty difficult to enter strings.

**Comment by Tanjitsu** [comment]
Is this based on a "clean" GTA with only ScriptHook + ScriptHookDotNet ?
Which version of ScriptHookDotNet ?

**Comment by kagikn** [comment]
Yeah, you're right, @tcpie. See [this line](https://github.com/crosire/scripthookvdotnet/blob/dev_v3/source/core/Console.cs#L529), and you'll realize the issue is true.

**Comment by crosire** [comment]
That should be easy to fix with:
```c#
if (e.Control)
    PasteClipboard();
else 
    AddToInput(GetCharsFromKeys(e.KeyCode, e.Shift, e.Alt));
```

**Comment by kagikn** [comment]
I know, we can fix this issue with the code crosire posted or with this (to follow DRY)
```c#
if (e.Control)
{
   PasteClipboard();
   break;
}
else 
    goto default;
```

**Comment by crosire** [comment]
Today I learned `goto` is a thing in C# ...

**Comment by FrederikBolding** [comment]
Made a tiny PR to fix the issue. I'm not sure you're allowed to use `goto default;` here @kagikn

**Comment by FrederikBolding** [comment]
I believe this is resolved. @tcpie I am unable to reproduce issues with the " character, but have fixed the issue with "V". Can you confirm?

**Comment by FrederikBolding** [comment]
I think this one can be closed @crosire

**Comment by kagikn** [comment]
Fixed in e376fb0, closing this issue.
